drivers/led_strip/led_strip_shell.c

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2025 Dmitrii Sharshakov <d3dx12.xx@gmail.com>
+ *
+ * Based on drivers/led/led_shell.c which is:
+ * Copyright (c) 2020 Seagate Technology LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "zephyr/kernel.h"
+#include "zephyr/sys/util.h"
+#include <zephyr/shell/shell.h>
+#include <zephyr/drivers/led_strip.h>
+#include <stdlib.h>
+
+#define LOG_LEVEL CONFIG_LOG_DEFAULT_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(led_shell);
+
+#define MAX_COLOR_ARGS 255
+
+enum {
+	arg_idx_dev	= 1,
+	arg_idx_value	= 2,
+};
+
+static int parse_common_args(const struct shell *sh, char **argv,
+			     const struct device **dev)
+{
+	*dev = shell_device_get_binding(argv[arg_idx_dev]);
+	if (!*dev) {
```

Comment:
No implicit boolean conversions.

```suggestion
	if (*dev == NULL) {
```